cmake/EthDependencies.cmake

```diff
@@ -1,21 +1,6 @@
 # all dependencies that are not directly included in the cpp-ethereum distribution are defined here
 # for this to work, download the dependency via the cmake script in extdep or install them manually!
 
-function(eth_show_dependency DEP NAME)
```

Comment:
This only works for old-style cmake dependencies - and we have none left after changing boost, so I'd say this can go.
Since we only include this file once, I also think the debug output below replacing this doesn't need to set and check global "DISPLAYED" properties like this here did.